src/Sarif/NotYetAutoGenerated/LogicalLocation.cs

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
```

Comment:
>using [](start = 0, length = 5)

If this file too is here just for the sake of applying the `DefaultValue` attribute, again, you don't need it -- use `AttributeHint`.

Reply:
[https://github.com/Microsoft/sarif-sdk/issues/1177](https://github.com/Microsoft/sarif-sdk/issues/1177)

---
In reply to: [243719690](https://github.com/Microsoft/sarif-sdk/pull/1170#discussion_r243719690) [](ancestors = 243719690)